docs/hu/SQL_Cheat_Sheet_magyar.md

```diff
@@ -0,0 +1,471 @@
+# Lekérdezés szintaktitka cheat-sheet
+
+Ebben a cheat-sheet-ben a különböző tanult nyelvek lekérdezési szintaktikája kerül összehasonlításra.
+
+**Fontos:** C# LINQ használatakor ne felejtsük el példányosítani az adatbázis kontextust egy  using block-ban.
+
+```csharp
+using (var db = new AdatvezDbContext())
+{
+    //lekérdezések, stb...
+}
+```
+
+## SELECT
+
+**SQL:**
+
+```SQL
+SELECT column1, column2
+FROM table
+WHERE condition;
+```
+
+**C# LINQ:**
+
+**Megjegyzés:** Select()-ben névtelen objektumot hoztunk létre, ennek attribútumai szabadon választhatóak, a vetítési művelet elvégzéséhez.
+```csharp
+//c# jellegű szintaktika
+var query = context.Table
+            .Where(item => condition)
+            .Select(item => new { item.Column1, item.Column2 });
+
+ //SQL szerű szintaktika           
+var query = from item in context.Table
+            where condition
+            select new { item.Column1, item.Column2 };
+```
+**FONTOS:** Ha szükségünk van a navigation propertykkel hivatkozott entitásokra, akkor a fejlesztő a kódban ezt specifikálja az Include használatával, és a rendszer betölti a kért hivatkozásokat is. Ezek attribútumait **NE** egyesével select segítségével adjuk vissza.
```

Comment:
Az utolsó mondatot nem értem. Valamikor valid, hogy a selectben adjuk meg a nav propertyket is